main/app.py

```diff
@@ -0,0 +1,352 @@
+from __future__ import print_function
+import random
+import sqlite3
+import sys
+import tkFileDialog
+from Tkinter import Tk
+
+from clint.textui import colored, puts
+from colorama import init, Back, Style  # Fore
+from pyfiglet import figlet_format
+from termcolor import cprint
+
+from .models.room import Office, LivingSpace
+from .util.File import FileParser
+
+
+class Amity(object):
+    """Description for amity"""
+
+    def __init__(self):
+        # variable used in create_rooms
+        self.rooms = {
+            'O': [],
+            'L': []
+        }
+        # variable used in add_person
+        self.people_data = {'Staff': [],
+                            'Fellow': []
+                            }
+
+
+        self.room_type = "O" or "L"
+
+        name = self.name = ""
+        personnel_type = self.personnel_type = ""
+        want_accommodation = self.want_accommodation = ""
+
+        self.offices = []
+        self.full_name = []
+        self.allocated_room = []
+
+        self.person_name = ""
+        self.person_type = ""
+        self.room_name = ""
+        self.people_in_room = []
+
+        self.person_name = []
+        self.person_type = ""
+
+        self.offices = []
+        self.housing = []
+
+        self.available_office = []
+        self.available_housing = []
+
+        self.office_allocations = []
+        self.living_alllocations = []
+        self.office_occupants = []
+        self.unallocated_offices = []
+        self.living_occupants = []
+        self.unallocated_houses = []
+        self.office = []
+        self.housing = []
+
+        self.living_occupied = 0
+        self.office_occupied = 0
+        file = self.file = " "
+        allocated = self.allocated = []
+
+        self.conn = sqlite3.connect("amity.sqlite")
+        self.conn.row_factory = sqlite3.Row
+        self.connect = self.conn.cursor()
+        self.connect.execute(
```

Comment:
**Code Quality: Nitpick**

Always run your code within a `try...except` block to take care of error cases.